src/rpc/blockchain.cpp

```diff
@@ -431,7 +432,9 @@ static RPCHelpMan getdifficulty()
 {
     return RPCHelpMan{"getdifficulty",
                 "\nReturns the proof-of-work difficulty as a multiple of the minimum difficulty.\n",
-                {},
+                {
+                    {"next", RPCArg::Type::BOOL, RPCArg::Default{false}, "difficulty for the next block, if found now"},
```

Comment:
@Sjors what do you mean by "if found now" here and the other places?

Reply:
In testnet3 and testnet4 the difficulty depends on the timestamp. After 20 minutes it drops to 1. These methods derive difficulty using the current timestamp.